### PR TITLE
monsters attack player

### PR DESCRIPTION
**Pull request by skylarjhdownes**
Monsters can attack the player using a getAttacked class.

### CODE REVIEW

File: source/index.js [hunk 1]
```diff
@@ -20,7 +21,13 @@ new Ninja()
 var lvl = getURLQuery("level") || 0
 for(var index in Levels[lvl].monsters) {
     var monster = Levels[lvl].monsters[index]
-    new Monster(monster)
+    if (monster.id === "grunt"){
+        new SkeletonGrunt(monster)
+    }
```
Comment (ehgoodenough) [inline source/index.js:26]: This should be an `else if`

File: source/scripts/models/SkeletonGrunt.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+var SkeletonGrunt = function(protomonster) {
+    this.id = ShortID.generate()
+    window.game.monsters[this.id] = this
+    
+    this.x = protomonster.x || protomonster.tx * 32 || 0
+    this.y = protomonster.y || protomonster.ty * 32 || 0
+    this.size = 48
+    this.speed = 1
+    this.currentAction = null
+    this.attackRange = 100
```
Comment (ehgoodenough) [inline source/scripts/models/SkeletonGrunt.js]: This variable is not used in SkeletonGrunt.

File: source/scripts/models/SkeletonGrunt.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    
+    if (this.currentAction && this.currentAction.moveTo){
+        if (this.currentAction.moveTo.y > this.y)
+            this.y += this.speed * delta
+        else 
+            this.y -= this.speed * delta
+        if (this.currentAction.moveTo.x > this.x)
+            this.x += this.speed * delta
+        else 
+            this.x -= this.speed * delta
```
Comment (ehgoodenough) [inline source/scripts/models/SkeletonGrunt.js:53]: Put curly braces here. :]

File: source/index.js (hunk 1 above)
Comment (skylarjhdownes) [inline source/index.js:26]: Do thing do.

**Pull request merged**